apiGenerator.ts

```diff
@@ -189,7 +189,7 @@ import {ExtendedRoutesConfig} from "@tsoa/cli/dist/cli";
         'controllerPathGlobs': ['./src/api/*'],
         'routesDir': './src/api/_auto',
         'authenticationModule': './src/middlewares/authentication.ts',
-        'noImplicitAdditionalProperties': 'silently-remove-extras' //Verify it
+        'noImplicitAdditionalProperties': 'ignore'
```

Comment:
Czy to tak wgl rozwiązuje cały ten problem 😝

Reply:
@waszkowski Tak, bo cały problem był w tym, że TOSA ucinała wszystkie pola bo ich nie znała, i finalnie pusty obiekt wchodził do use casu xd